test/schema-feature.test.js

```diff
@@ -982,42 +982,71 @@ test('Check how many AJV instances are built #1', t => {
   fastify.ready(err => {
     t.error(err)
 
-    t.ok(fastify.validatorCompiler, 'validator initlialized on preReady')
+    t.ok(fastify.validatorCompiler, 'validator initialized on preReady')
     fastify.validatorCompiler.checkPointer = true
     instances.forEach(i => {
-      t.ok(i.validatorCompiler, 'validator initlialized on preReady')
+      t.ok(i.validatorCompiler, 'validator initialized on preReady')
       t.equals(i.validatorCompiler.checkPointer, true, 'validator is only one for all the instances')
     })
   })
 })
 
-test('Check how many AJV instances are built #2', t => {
-  t.plan(7)
+test('Check how many AJV instances are built #2 - verify validatorPool', t => {
+  t.plan(13)
   const fastify = Fastify()
-  t.notOk(fastify.validatorCompiler, 'validator not initlialized')
+  t.notOk(fastify.validatorCompiler, 'validator not initialized')
 
+  // SIBLING 1
```

Comment:
```suggestion
  // Sibling 1
```

Overall avoid CAPS LOCK :).

Reply:
fixed

Dear, old, function's name 😂